src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
```

Comment:
You can make Random a class-level variable. Check [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)
```suggestion
    Random random = new Random();
```